### PR TITLE
fix(payload): Passes correct path to import modules on Windows started with file://

### PR DESCRIPTION
**Pull request by r1tsuu**
## Description

TL;DR: There's a Discord thread with that issue https://discord.com/channels/967097582721572934/1225824784609316906/1227622673564041370
Here's what was wrong https://github.com/nodejs/node/issues/31710
It's working fine for me with `generate:types` though. But when i'm trying to run a standalone script (from demo repo) - not, i'm getting this:
```
Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file, data, and node are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/load:239:11)
    at defaultLoad (node:internal/modules/esm/load:130:3)
    at nextLoad (node:internal/modules/esm/hooks:866:28)
    at Z (file:///C:/Users/enjoy/work/payload/node_modules/.pnpm/tsx@4.7.2/node_modules/tsx/dist/esm/index.mjs:5:1618)
    at nextLoad (node:internal/modules/esm/hooks:866:28)
    at Hooks.load (node:internal/modules/esm/hooks:449:26)
    at ModuleLoader.load (node:internal/modules/esm/loader:402:34)
    at ModuleLoader.moduleProvider (node:internal/modules/esm/loader:285:56)
    at new ModuleJob (node:internal/modules/esm/module_job:67:26)
    at #createModuleJob (node:internal/modules/esm/loader:297:17) {
  code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
}
```
I added to `_community` folder a script (probably should remove it), you can run it like this
`pnpm tsx .\test\_community\standalone-script.ts`
It should throw `ValidationError` email already exists, but it's fine. I tested it on a normal project (macOS and Windows) and it's working as expected.

- [x] I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.

### CODE REVIEW

**Comment by paulpopus** [comment]
Assigning to @DanRibbens since it looks like a windows thing

**Comment by DanRibbens** [comment]
I tested this locally and it worked from the demo project, whereas before I was getting the same errors. Great work @r1tsuu!

**Comment by r1tsuu** [comment]
I removed the script in tests and a bit updated variables assigning i think it's better looking now.

**Review by DanRibbens — APPROVED** [review]

**Comment by DanRibbens** [comment]
Awesome! I'll merge this once it gets through CI.
